src/services/API.js

```diff
@@ -13,12 +22,23 @@ function signUpErrorAlert(error) {
     };
 }
 
+function logIn({ requestBody, history, setIsLoading }) {
+    setIsLoading(true);
+    const request = axios.post(API_URL + SIGNIN, requestBody);
```

Comment:
Não seria melhor colocar direto "/sign-up" ou  "/sign-in" ao inves de criar variaveis pra isso?

Reply:
Pensei em fazer assim pra se por acaso mudar tá fácil...(aqui provavelmente não vai mas numa situação real talvez).
@brunomguimaraes O que pensa sobre isso?

Reply:
Eu faria a parte de `SIGIN`  direto neste caso e chamaria a função de `signIn`